src/components/POIMarker.component.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+class POIMarker extends Component {
+  constructor(props) {
+    super(props)
+    this.onPOIClick = this.onPOIClick.bind(this)
+  }
+
+  onPOIClick(event) {
+    // Modify the App's state: set selectedEvent to be this POI
+    this.props.onPOIMarkerClick(this.props.id)
+  }
+
+  render() {
+    const xC = `${this.props.coordinateX / 100 * this.props.mapImageWidth}px`
+    const yC = `${this.props.coordinateY / 100 * this.props.mapImageHeight}px`
+    return (
+      <div>
+        <div
+          className={
+            this.props.isSelected ? 'selected-marker' : 'active-marker'
```

Comment:
Now that we want to use the BEM style names, we will need something like this:
```javascript
className={
  this.props.isSelected ? 'marker marker--selected' : 'marker'
}
```
Notice that if it's selected, we need to give both `marker` _and_ `marker--selected` classes.

To make this cleaner, we can use the [`classnames` library](https://github.com/JedWatson/classnames). 

This will allow us to conditionally set class names by doing something like this:
```javascript
className={
  classnames('marker', {
    'marker--selected': this.props.isSelected
  })
}
```

But make sure you add
```javascript
import classnames from 'classnames'
```
at the top first.